crates/biome_css_analyze/src/utils.rs

```diff
@@ -109,3 +110,21 @@ pub fn is_function_keyword(value: &str) -> bool {
 pub fn is_custom_function(value: &str) -> bool {
     value.starts_with("--")
 }
+
+// Returns the vendor prefix extracted from an input string.
+pub fn vender_prefix(prop: &str) -> String {
+    let prefixes = ["-webkit-", "-moz-", "-ms-", "-o-"];
```

Comment:
These prefixes should move to keyword.rs

Reply:
Oops, I'm sorry.
I fixed it with the following commit.
808cc3711b2abb7f791eed97996ae70ab39cc7c0